spec/services/claim_fast_tracking/max_rating_annotator_spec.rb

```diff
@@ -61,6 +61,56 @@
         end
       end
 
+      context 'when a disabilities response contains unexpected data' do
+        context 'disabilities response contains nil for rated_disabilities' do
+          let(:rated_disabilities) do
+            nil
+          end
```

Comment:
Short expressions in the block can be collapsed down to a one-liner: `let(:rated_disabilities) { nil }`